### DISCUSSION TITLE
Invert thumbstick for play with Twilight Princess GameCube

### DISCUSSION THREAD

**Discussion by mike84-nl** [darthcloud/BlueRetro#1116] [Q&A]
Wondering if there's a way to possibly invert the right thumbstick on the horizontal axis within the web interface for my Blueretro? always wanted to play Twilight Princess on GameCube but just cant get use the the camera setting making the game unplayable for me. I know i can emulate the game but wanted to pay on my GameCube and my CRT. Any help greatly appreciated Thx

**Comment by darthcloud** [comment]
In the mapping config just map joystick Up as down and down as up
